### PR TITLE
016/master

### PR DESCRIPTION
**Pull request by BenGitsCode**
Update from diagnostic-template

Refresh for cohort 016

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ so much for us?
 // your response here
 ```
 
-What does typing `rails console` in a directory that houses a rails app do for
+What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
```
Comment (raq929) [inline diagnostic.md:57]: This should be `bin/rails console` I think.

Comment (BenGitsCode) [reply diagnostic.md:57]: I thought so too and Antony said he prefers `bin/rails dbconsole` here https://github.com/ga-wdi-boston/rails-api-single-diagnostic/issues/17#issuecomment-278048650

Comment (raq929) [reply diagnostic.md:57]: I think we mostly use rails console in the lesson, so we should use it here.

Comment (jrhorn424) [reply diagnostic.md:57]: In fact, I did use `bin/rails db` but elected to stay away from `bin/rails console` until `rails-api-one-to-many`.

I'm not sure that's the way it should be. But it's the way it was done.

If both need to be covered, they should have learning objectives.

**Comment by jrhorn424** [comment]
@BenGitsCode This update was missing the comfort/clarity section, which I walked you through updating last week. Please ensure it is included next time. Thanks!

See #46

**Pull request merged**